dpc-api/src/main/resources/prod.application.env

```
@@ -0,0 +1,7 @@
+PUBLIC_URL= "https://prod.dpc.cms.gov/api"
```

This has an extra space which will break the sourcing 😄 (tested it locally)

```
./prod.application.env:1: no such file or directory: https://prod.dpc.cms.gov/api
```

```suggestion
PUBLIC_URL="https://prod.dpc.cms.gov/api"
```